exercises/js/blog.js

```diff
@@ -0,0 +1,24 @@
+class RevealText {
+  constructor(listSelector) {
+    this.blogList = $(listSelector);
+  }
+
+  init() {
+    this.bindClickTolist();
+  }
+
+  bindClickTolist() {
+    this.blogList.bind("click", function(event) {
+      event.preventDefault(); 
+      let h3Element = $(event.target.closest("h3"));
+      if(h3Element) {
```

Comment:
To avoid such checks pass `h3` elements instead of `ul` to the class

Reply:
I have used ul so that I could use event delegation. If I had used h3 elements then I have to attach click listener to each and every h3 element.

Reply:
@sgoyal12 Click event can be attached on `$("#blog h3")`